go.mod

```diff
@@ -1,19 +1,19 @@
 module github.com/GoogleCloudPlatform/golang-samples
 
-go 1.11
+go 1.14
```

Comment:
Keep as 1.11. This signifies the Go version compatibility, which we need to keep at 1.11, the oldest supported GAE/GCF version.